components/sidebar/card/card-section/CardSection.tsx

```diff
@@ -22,7 +24,7 @@ const CardSection = ({
         "bg-card": !isOpen,
       })}
     >
-      <a href={href} className="flex items-center gap-4">
+      <a href={href} className="flex items-center gap-4" onClick={onClick}>
```

Comment:
you should make the onClick for the whole div, not only the icon, we should be able to log out when click on the title too